TikTokApi/tiktok.py

```diff
@@ -1027,8 +1045,14 @@ def getUser(self, username, **kwargs) -> dict:
 
         t = r.text
 
-        j_raw = t.split('<script id="__NEXT_DATA__" type="application/json" crossorigin="anonymous">')[1].split("</script>")[0]
-         
+        try:
+            j_raw = t.split('<script id="__NEXT_DATA__" type="application/json" crossorigin="anonymous">')[1].split("</script>")[0]
+        except IndexError :
+            if (len(t) == 0):
```

Comment:
```suggestion
            if not t:
```
empty strings are falsy in python, and this would also handle the case where `t` is `None`